src/Bridge/Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -318,4 +308,23 @@ private function getFilterValueFromUrl(string $value) : string
 
         return $value;
     }
+
+   /**
+    * Work on the values array.
+    *
+    * @param array $values
+    *
+    * @return array
+    */
+   private function getValuesFromArray(array $values) : array
+   {
+       foreach ($values as $k => $v) {
```

Comment:
I'm not super fond of one single char variables: `$key`, `$value` would be better IMO even if it's not super useful